_layouts/base.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: 'en-US' }}">
+  <head>
+    {% include meta.html %}
+    {% include styles.html %}
+    {% include google_analytics.html %}
+    {% include dap.html %}
+    {% include google_recaptcha.html %}
```

Comment:
We didn't previously include this in the 404 layout. Should we conditionally include it? I doubt it hurts. TBH, it should probably only be added as an extra include on the contact form page, since I think that's the only place we're actually using captcha.

But now that I think about it, if we're only using it on the one page, yet include it in the page layout on every other page, clearly it should also not be a problem to include in the 404 layout.

🤷

Reply:
Yeah my thinking was the same -- if it's not causing any harm on non-contact pages, it's probably fine where it is